spec/mercure.md

```diff
@@ -324,6 +323,45 @@ after a long disconnection time).
 The hub **MAY** also specify the reconnection time using the `retry` key, as specified in the
 server-sent events format.
 
+# Subscription Events
+
+The hub **MAY** publish an update when a subscription to a topic is created or terminated. If this
+feature is implemented by the hub, an update **MUST** be dispatched every time that a subscription
+is created or terminated, and for each topic to which the client subscribes.
+
+The topic of this update **MUST** follow the pattern
+`https://mercure.rocks/subscriptions/{topic}/{subscriptionID}` where `topic` is the URL-encoded
+value of the subscribed topic and `subscriptionID` is an unique identifier for this subscription.
+`subscriptionID` **MAY** be a UUID [@RFC4122].
+
+The content of the update **MUST** be a JSON-LD [@!W3C.REC-json-ld-20140116] document containing at
+least the following properties:
+
+ *  `@id`: the identifier of this update, it **MUST** be the same value as the subscription update's
+    topic
+
+ *  `@type`: the fixed value `https://mercure.rocks/Subscription`
```

Comment:
I'm not sure I agree that this should be a fixed value, it should at least be configurable. I mean what happens if you want to change the name, or you get hit by a bus and we can't get the domain?

Reply:
See my previous comment. It's not related to hostnames, domains and the like. It's a RDF type. It's similar to a XML namespace. The value never changes, and doesn't have to be dereferencable (the URL may not exist).